src/main/java/frc/robot/subsystems/StorageSubsystem.jav

```diff
@@ -0,0 +1,25 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.SubsystemBase;
+import edu.wpi.first.wpilibj.VictorSP;
+import frc.robot.Constants;
+
+public class StorageSubsystem extends SubsystemBase {
+  private final VictorSP StorageGate;
+  private final VictorSP StorageMid;
+
+  public StorageSubsystem() {
+    StorageMid = new VictorSP(Constants.STORAGE_MID_PWM_CHAN);
+    StorageGate = new VictorSP(Constants.STORAGE_GATE_PWM_CHAN);
+  }
+  public runMid() {
+    StorageMid.set(1);
+  }
+  public runGate() {
+    StorageGate.set(0.1);
+  }
+  public stopMotors() {
+    StorageGate.stopMotors();
```

Comment:
You'll likely want to be able to keep the mid running while stopping the gate.  I would recommend an API more along the lines of:
public void runMid(bool run);
public void runGate(bool run);
public void stopAll();